tests/e2e/linalg/CMakeLists.txt

```diff
@@ -93,3 +97,40 @@ iree_check_single_backend_test_suite(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+if(IREE_ENABLE_ASAN)
+iree_check_single_backend_test_suite(
+  NAME
+    check_llvm-cpu_local-task_asan
+  SRCS
+    "conv2d.mlir"
+    "f32_to_i4.mlir"
+    "i4_to_f32.mlir"
+  TARGET_BACKEND
+    "llvm-cpu"
+  DRIVER
+    "local-task"
+  COMPILER_FLAGS
+    "--iree-llvmcpu-link-embedded=false"
+    "--iree-llvmcpu-sanitize=thread"
```

Comment:
oops, fixing this to `address`

logs before fix: https://github.com/openxla/iree/actions/runs/7937560045/job/21674988323?pr=16452